lib/utils/commands.js

```diff
@@ -9,5 +9,7 @@ const names = commands
     })
     .reduce((arr, val) => arr.concat(val), []);
 
-module.exports.names = names;
-module.exports.info = commands;
+module.exports = {
```

Comment:
reassigning the `exports` object should be avoided.
Please revert this change.

Reply:
Changed the named export to `module.exports.commands = commands;`